sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -546,33 +545,54 @@ private void processPendingErrors() {
             }
         }
 
-        if (shouldStopProcessingPendingErrors()) {
-            return;
+        /* If automatic processing is enabled. */
+        if (mAutomaticProcessing) {
+
+            /* Proceed to check if user confirmation is needed. */
+            sendCrashReportsOrAwaitUserConfirmation();
         }
 
-        processUserConfirmation();
     }
 
-    private void processUserConfirmation() {
+    /**
+     * Check user confirmation or send logs now.
```

Comment:
Can you add `@return` here? I don't know the purpose of the return value. Is it only for testing?

Reply:
I don't put Javadoc for private and package private. The JavaDoc can be found in WrapperSdkExceptionManager.

Reply:
This is why we need java doc for this method even this is a private method.
https://github.com/Microsoft/mobile-center-sdk-android/pull/526#discussion_r144171645
Can you just say here that the method returns an alwaysSend flag?